cr8/clients.py

```diff
@@ -172,6 +173,34 @@ def _to_dsn(hosts):
     return dsn
 
 
+def _to_boolean(v):
+    if str(v).lower() in ("yes", "true", "t", "1"):
+        return True
+    elif str(v).lower() in ("no", "false", "f", "0"):
+        return False
+    else:
+        raise ValueError('not a boolean value')
+
+
+def _verify_ssl(hosts):
```

Comment:
Maybe rename to `_verify_ssl_from_first` to clarify that it takes the value of the first explicitly specified `verify_ssl` flag?